<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWindow</class>
 <widget class="QMainWindow" name="GraphWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QChartView" name="GraphA_4" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="QChartView" name="GraphA_3" native="true"/>
    </item>
    <item row="0" column="0">
     <widget class="QChartView" name="GraphA" native="true"/>
    </item>
    <item row="0" column="1">
     <widget class="QChartView" name="GraphA_2" native="true"/>
    </item>
    <item row="2" column="0">
     <widget class="QChartView" name="GraphA_5" native="true"/>
    </item>
    <item row="2" column="1">
     <widget class="QChartView" name="GraphA_6" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>QChart/QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
